src/locale/languages.ts

```diff
@@ -51,6 +52,7 @@ export const APP_LANGUAGES: AppLanguageConfig[] = [
   {code2: AppLanguage.ko, name: '한국어 – Korean'},
   {code2: AppLanguage.pt_BR, name: 'Português (BR) – Portuguese (BR)'},
   {code2: AppLanguage.ru, name: 'Русский – Russian'},
+  {code2: AppLanguage.ro, name: 'Română – Romanian'},
```

Comment:
I have a tiny suggested tweak here: would it be possible to keep the languages in alphabetical order by `code2` and put the Romanian entry above the Russian one?

Reply:
Fixed. Thanks